src/entities/index.ts

```diff
@@ -1,5 +1,6 @@
 export { default as AssetTypes, AssetType } from './assetTypes';
 export type { TxFee, SelectedGasPrice } from './gas';
+export type { NativeCurrencyKeys } from './nativeCurrencyTypes';
```

Comment:
```export { NativeCurrencyKeys } from './nativeCurrencyTypes';```

Reply:
I just happened to run this locally while trying to test for something else and saw "cannot find USD of undefined" - which was very confusing! but removing "type" here brings back the `NativeCurrencyKeys` mapping!

Reply:
I wonder whether `NativeCurrencyKeys` should be renamed to `nativeCurrencyKeys` to avoid confusion since title case makes it look like a type. Don't need to block this PR over it though.